django/contrib/postgres/fields/ranges.py

```diff
@@ -98,6 +98,38 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class RangeContainedBy(models.Lookup):
+    lookup_name = 'contained_by'
+    type_mapping = {
+        'integer': 'int4range',
+        'bigint': 'int8range',
+        'double precision': 'numrange',
+        'date': 'daterange',
+        'timestamp with time zone': 'tstzrange',
+    }
+
+    def as_sql(self, qn, connection):
+        field = self.lhs.output_field
+        if isinstance(field, models.FloatField):
```

Comment:
I don't think it's worth it here... Sadly postgres is really fussy about what data types we use. It's worth noting that any subclass of `FloatField` which doesn't change the `db_type` will work here. Anything else you'd need to write a custom transform anyway.